### PR TITLE
[BACKLOG-32232] Fix minor issues in File Open/Save Dialog

### PR DESCRIPTION
**Pull request by bmorrise**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
:ok_hand: All tests passed!
 
Tests run: **8**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7025/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
:ok_hand: All tests passed!
 
Tests run: **8**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7025/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/file-open-save-new/core
```
---
:ok_hand: All tests passed!
 
Tests run: **8**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7025/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by bmorrise** [comment]
@ddiroma

File: plugins/file-open-save-new/core/src/main/resources/i18n/file-open-save-new/messages.properties [hunk 1]
```diff
@@ -136,4 +136,7 @@ file-open-save-plugin.moving.message=Moving Files...
 
 file-open-save-plugin.close.button=Close
 file-open-save-plugin.vfs.unable-to-connect.title=Unable to connect
-file-open-save-plugin.vfs.unable-to-connect.message=We were unable to connect to selected VFS location. Please check the path you're using and make sure the server is up and running.
\ No newline at end of file
+file-open-save-plugin.vfs.unable-to-connect.message=We were unable to connect to selected VFS location. Please check the path you're using and make sure the server is up and running.
```
Comment (ddiroma) [inline plugins/file-open-save-new/core/src/main/resources/i18n/file-open-save-new/messages.properties]: does the apostrophe need to be double... ''

Comment (ddiroma) [reply plugins/file-open-save-new/core/src/main/resources/i18n/file-open-save-new/messages.properties]: Also, the one below and the one in the other messages file

**Pull request merged**